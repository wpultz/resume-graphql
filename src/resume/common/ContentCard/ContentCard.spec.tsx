import * as React from 'react'
import renderer from 'react-test-renderer'

import { ContentCard } from './index'

describe('ContentCard component', () => {
  describe('Snapshots', () => {
    it('should render with string content', () => {
      const snap = renderer.create(<ContentCard title="This little piggy" content="Went to market" />).toJSON()
      expect(snap).toMatchSnapshot()
    })

    it('should render with React content', () => {
      const snap = renderer
        .create(<ContentCard title="This little piggy" content={<div>Ate roast beef</div>} />)
        .toJSON()
      expect(snap).toMatchSnapshot()
    })
  })
})
