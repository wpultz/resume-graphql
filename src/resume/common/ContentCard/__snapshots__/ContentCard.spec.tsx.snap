// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentCard component Snapshots should match snapshot 1`] = `
<div>
  <h1>
    Big Wolf
  </h1>
  <div
    className="content"
  >
    On campus
  </div>
</div>
`;

exports[`ContentCard component Snapshots should render with React content 1`] = `
<div>
  <h1>
    This little piggy
  </h1>
  <div
    className="content"
  >
    <div>
      Ate roast beef
    </div>
  </div>
</div>
`;

exports[`ContentCard component Snapshots should render with string content 1`] = `
<div>
  <h1>
    This little piggy
  </h1>
  <div
    className="content"
  >
    Went to market
  </div>
</div>
`;
